### PR TITLE
Feat/rules

### PR DESCRIPTION
**Pull request by Chinimala**

### CODE REVIEW

File: packages/components/Rules/GameGuides/index.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      <AccordionButton>
+        <Heading size="l" flex="1" textAlign="left">
+          Exemples de comportements strictement interdits :
+        </Heading>
+        <AccordionIcon />
+      </AccordionButton>
+      <AccordionPanel pb={4}>
+        <RulesList contents={forbidden} icon={MdCircle} iconColor="red.400" />
+      </AccordionPanel>
+    </AccordionItem>
```
Comment (Tripouille) [inline packages/components/Rules/GameGuides/index.tsx]: Maybe we should create a component ?

File: packages/components/Rules/GameRules/RuleMultipleText/index.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+const RuleMultipleText: React.FC<RuleMultipleTextProps> = ({ content, className }) => (
+  <>
+    {content.map((text, index) => (
+      // eslint-disable-next-line react/no-array-index-key
+      <Text key={index} mb="2" className={className}>
+        {text}
+      </Text>
+    ))}
+  </>
+);
```
Comment (Tripouille) [inline packages/components/Rules/GameRules/RuleMultipleText/index.tsx]: Can you add comments on this part?

File: packages/components/Rules/index.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+          <TabPanel>
+            <DiscordRules />
+          </TabPanel>
+          <TabPanel>
+            <GameRules />
+          </TabPanel>
+          <TabPanel>
+            <GameGuides />
+          </TabPanel>
+        </TabPanels>
```
Comment (Tripouille) [inline packages/components/Rules/index.tsx]: Maybe consider to use map?

**Review by Tripouille — CHANGES_REQUESTED** [review]

File: packages/components/Rules/GameGuides/index.tsx (hunk 1 above)
Comment (Chinimala) [reply packages/components/Rules/GameGuides/index.tsx]: Component "CustomAccordionItem" created

Comment (Chinimala) [reply packages/components/Rules/GameGuides/index.tsx]: Component "CustomAccordionItem" created

File: packages/components/Rules/GameRules/RuleMultipleText/index.tsx (hunk 2 above)
Comment (Chinimala) [reply packages/components/Rules/GameRules/RuleMultipleText/index.tsx]: Component "CustomText" created with comments

File: packages/components/Rules/index.tsx (hunk 3 above)
Comment (Chinimala) [reply packages/components/Rules/index.tsx]: Done, thank you ! ❤️

**Review by Tripouille — APPROVED** [review]

**Pull request merged**